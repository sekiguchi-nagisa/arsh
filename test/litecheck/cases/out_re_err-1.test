
# RUN:   /bin/sh $self
echo hello success!!
echo AAA ZZZ BBB
# CHECK_RE: hello s.+
# CHECK_RE: .+ C+