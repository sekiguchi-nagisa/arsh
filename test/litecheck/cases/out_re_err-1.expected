CHECK_RE on line 5,
    Expect: .+ C+
    Actual: AAA ZZZ BBB
stdout is mismatched